content/docs/editing/overview.mdx

```diff
@@ -19,10 +19,11 @@ date: '2021-01-01T00:00:00.000Z'
 Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi ultricies urna ut ex varius, sed fringilla nibh posuere. Vestibulum a pulvinar eros, vel varius orci. Sed convallis purus sed tellus pellentesque ornare quis non velit. Quisque eget nibh nec nisl volutpat aliquet. Donec pharetra turpis vitae diam aliquam rutrum. Sed porta elit ut mi vehicula suscipit. Ut in pulvinar nunc.
 ```
 
-![markdown-editing](https://res.cloudinary.com/forestry-demo/image/upload/v1645712826/tina-io/docs/markdown.gif)
+<WebmEmbed embedSrc="/img/docs/editing/markdown.webm"/>
```

Comment:
**issue**: The spacing around this image needs to be corrected, as it is colliding with the text block above it.

![image](https://github.com/user-attachments/assets/7d0f86f2-0f2b-4723-b370-221ae34a49f1)
**Figure: There is no spacing above the updated image on the left, where there was previously**

Reply:
Resolved - i also added some context for the video